src/bionics.cpp

```diff
@@ -546,11 +547,28 @@ bool player::activate_bionic( int b, bool eff_only )
         if( !has_cable ) {
             add_msg_if_player( m_info,
                                _( "You need a jumper cable connected to a vehicle to drain power from it." ) );
+        } else {
+            const optional_vpart_position vp_pos = g->m.veh_at( pos() );
+            if( vp_pos ) {
```

Comment:
You can make these three lines into one:
```C++
} else if( const optional_vpart_position vp_pos = g->m.veh_at( pos() ) ) {
```

You can even include the extraction of the seat part:
```C++
} else if( const cata::optional<vpart_reference> seat = g->m.veh_at( pos() ).part_with_feature( "BOARDABLE" ) ) {
```

Reply:
I think I was being overly cautious. Fixed.